sdk/cognitiveservices/cognitiveservices-qnamaker/src/qnAMakerClient.ts

```diff
@@ -28,8 +28,8 @@ class QnAMakerClient extends QnAMakerClientContext {
    * @param credentials Subscription credentials which uniquely identify client subscription.
    * @param [options] The parameter options
    */
-  constructor(endpoint: string, credentials: msRest.ServiceClientCredentials, options?: msRest.ServiceClientOptions) {
-    super(endpoint, credentials, options);
+  constructor(credentials: msRest.ServiceClientCredentials, endpoint: string, options?: msRest.ServiceClientOptions) {
+    super(credentials, endpoint, options);
```

Comment:
There are breaking changes in the client constructor. Hence bumping the version to 2.0.0